### ISSUE TITLE
Uncaught TypeError: Cannot read property 'forEach' of null

### ISSUE THREAD

**Issue by sglyon** [dan12mol/toggl#1]
[Enter steps to reproduce below:]
1. Open editor
2. Use `cmd+ctrl+s` keybinding to start toggl

**Atom Version**: 0.209.0
**System**: Mac OS X 10.10.3
**Thrown From**: [toggl](https://github.com/dan12mol/toggl) package, v0.5.2
### Stack Trace

Uncaught TypeError: Cannot read property 'forEach' of null

```
At /Users/sglyon/.atom/packages/toggl/lib/toggl.js:47

TypeError: Cannot read property 'forEach' of null
  at /Users/sglyon/.atom/packages/toggl/lib/toggl.js:47:25
  at Request.onresponse [as _callback] (/Users/sglyon/.atom/packages/toggl/node_modules/toggl-api/lib/client.js:217:7)
  at Request.self.callback (/opt/homebrew-cask/Caskroom/atom/latest/Atom.app/Contents/Resources/app.asar/node_modules/request/request.js:123:22)
  at emitTwo (events.js:87:13)
  at Request.emit (events.js:169:7)
  at Request.<anonymous> (/opt/homebrew-cask/Caskroom/atom/latest/Atom.app/Contents/Resources/app.asar/node_modules/request/request.js:1047:14)
  at emitOne (events.js:82:20)
  at Request.emit (events.js:166:7)
  at IncomingMessage.<anonymous> (/opt/homebrew-cask/Caskroom/atom/latest/Atom.app/Contents/Resources/app.asar/node_modules/request/request.js:998:12)
  at emitNone (events.js:72:20)
  at IncomingMessage.emit (events.js:163:7)
  at _stream_readable.js:891:16
  at process._tickCallback (node.js:357:13)

```
### Commands

```
     -0:40.1.0 command-palette:toggle (div.settings-view.pane-item)
     -0:38.6.0 core:move-down (atom-text-editor.editor.mini.is-focused)
     -0:38.3.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -0:38.1.0 Toggl:Start (div.settings-view.pane-item)
  3x -0:36.4.0 core:move-down (input#togglInput.form-control)
  2x -0:35.8.0 core:move-up (input#togglInput.form-control)
     -0:35.4.0 core:move-right (input#togglInput.form-control)
  5x -0:34.7.0 core:move-down (select#projectSelect.form-control)
     -0:12.4.0 pane:show-previous-item (div.settings-view.pane-item)
     -0:08.4.0 command-palette:toggle (atom-text-editor.editor.vim-mode.command-mode.is-focused)
  3x -0:07.2.0 core:backspace (atom-text-editor.editor.mini.is-focused)
     -0:05.2.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -0:05.2.0 Toggl:Start (atom-text-editor.editor.vim-mode.command-mode)
     -0:03.5.0 vim-mode:change-to-last-character-of-line (atom-text-editor.editor.vim-mode.command-mode.is-focused)
     -0:01.5.0 core:undo (atom-text-editor.editor.vim-mode.is-focused.insert-mode.autocomplete-active)
```
### Config

``` json
{
  "core": {
    "themes": [
      "isotope-ui",
      "tomorrow-night-eighties"
    ],
    "projectHome": "/Users/sglyon",
    "disabledPackages": [
      "slack-atom",
      "symbols-view",
      "atom-cli-diff",
      "wrap-guide",
      "slack-chat",
      "build",
      "asciidoc-preview",
      "regex-railroad-diagram",
      "autocomplete-haskell",
      "haskell-ghc-mod",
      "ide-haskell",
      "language-haskell",
      "language-scala",
      "scala-format",
      "foldingtext-for-atom",
      "color-picker",
      "todo-list"
    ]
  },
  "toggl": {
    "togglApiKey": "9e9a3d83009e25b89a924dc1762efba7"
  }
}
```
### Installed Packages

``` coffee
# User
atom-alignment, v0.12.1
atom-ctags, v4.1.2
atom-live-grammar-reload, v0.3.0
atom-spotify2, v2.1.2
atomatigit, v1.5.3
autocomplete-emojis, v2.2.2
autocomplete-paths, v1.0.2
autocomplete-plus-python-jedi, v0.3.6
autocomplete-python, v0.4.2
file-icons, v1.5.7
fonts, v0.4.2
git-tab-status, v1.9.2
isotope-ui, v2.4.0
language-asciidoc, v0.11.1
language-julia, v0.0.0
language-latex, v0.6.1
language-matlab, v0.1.1
language-r, v0.2.0
language-swift, v0.4.0
last-cursor-position, v0.9.0
latex, v0.24.1
latexer, v0.2.3
linter, v0.12.7
linter-flake8, v1.4.2
markdown-preview-pandoc, v0.0.14
minimap, v4.10.0
multi-wrap-guide, v0.17.1
pdf-view, v0.22.0
project-manager, v1.15.10
python-yapf, v0.7.0
time-status, v1.1.0
toggl, v0.5.2
tomorrow-night-eighties, v0.4.0
tool-bar, v0.1.7
vim-mode, v0.53.0
wakatime, v4.0.9

# Dev
No dev packages
```

**Comment by dan12mol** [comment]
Fixed by [v0.6.0](https://github.com/dan12mol/toggl/releases/tag/v0.6.0)

**Comment by sglyon** [comment]
quick work, thanks!
